packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/NotebookSessionController.java

```diff
@@ -88,7 +87,7 @@ ResponseEntity<List<NotebookSession>> getNotebookSessions(
 	/**
 	 * Create session and return its ID
 	 *
-	 * @param request session to create and projectId
+	 * @param session session to create and projectId
```

Comment:
Why the double `session`?

Reply:
To match the method definition - at some point someone changed the parameter name from `request` to `session`.  Github doesn't do the nice formatting but the first word after param is the variable name, then the next string is the description: 
![image](https://github.com/DARPA-ASKEM/terarium/assets/30270901/557743bb-fefe-4239-904f-41e0d7b49d11)